### PR TITLE
remove call to GetDevice in ssh exec

### PR DESCRIPTION
**Pull request by heiytor**

### CODE REVIEW

File: ssh/server/handler/ssh.go [hunk 1]
```diff
@@ -404,15 +404,6 @@ func exec(api internalclient.Client, sess *session.Session, device *models.Devic
...
 
-	dev, err := api.GetDevice(device.UID)
-	if err != nil {
-		log.WithError(err).
-			WithFields(log.Fields{"client": uid}).
-			Error("failed to get device")
-
-		return err
-	}
-
```
Comment (otavio) [inline ssh/server/handler/ssh.go:415]: Split the rest of code in another commit.

**Review by otavio — CHANGES_REQUESTED** [review]

**Review by gustavosbarreto — APPROVED** [review]

**Review by henrybarreto — APPROVED** [review]

**Review by otavio — APPROVED** [review]
